create_infotexts.py

```diff
@@ -198,37 +198,47 @@ def main():
     :metadata_json: created with script `metadata_to_json_and_fnamesmap.py
     """
     metadata_json = "SMVK-Cypern_2017-01_metadata.json"
-    outpath = "./infofiles/"
+    outfile = open("./SMVK-Cypern_2017-02_wikiformat_data.json", "w")
 
     # Hack to printout a wikitable to copy-paste to WikiCommons
     people = create_people_mapping_wikitable(people_mapping)
     # print(people + "\n")
 
     metadata = load_json_metadata(metadata_json)
+    batch_dict = {}
     for fotonr in metadata:
+        batch_dict[fotonr] = {}
```

Comment:
1. added img_info = {}. Also changed `batch_dict` to `batch_info` for consistency. Done.
2. The original filename is the `fotonr` (plus extension) in this batch.